examples/nivisa_dmm_measurement/_helpers.py

```diff
@@ -100,13 +65,7 @@ def resolve_file_path(self, file_path: str) -> str:
         """
         if pathlib.Path(file_path).is_absolute():
             return file_path
-        (
-            _,
-            absolute_path,
-            _,
-            _,
-            user_canceled,
-        ) = self._sequence_context.Engine.FindFileEx(
+        (_, absolute_path, _, _, user_canceled) = self._sequence_context.Engine.FindFileEx(
             fileToFind=file_path,
```

Comment:
is this indentation right?

Reply:
Yes, this is how black indents the code with `line-length = 100`. The other way is how black indents the code with the default `line-length`.